operations/README.md

```diff
@@ -157,6 +157,12 @@ This section describes all operations supported by the API, organised here by th
 | :-- | :-- |
 | [Get all service order notes](serviceorders.md#get-all-service-order-notes) | Returns all notes associated with the given service orders. Service orders can be reservations or product orders. |
 
+## Order items
```

Comment:
The new operation should be listed under the **Finance** section, with all the other operations related to accounting items, payments, orders and bills. That is instead of creating a new section called **Order items**.